typescript/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "example-applications-typescript",
+  "version": "1.0.0",
+  "description": "Typescript example",
+  "scripts": {
+    "test": "mocha",
+    "compile": "tsc"
+  },
```

Comment:
Should include "directories.lib" and "directories.test" entries.

Reply:
Think tests are inside src here so will leave directories.tests out.

Reply:
Yeah, but that's _what_ you're attempting to communicate...